lib/compilers/argument-parsers.js

```diff
@@ -135,6 +135,12 @@ class ClangParser extends BaseParser {
     }
 }
 
+class PythonParser extends BaseParser {
```

Comment:
Not sure if this is actually needed - it may be better to use `BaseParser` in the python compiler.

Looking through the output of `python3 --help` I don't see many options that actually apply.
Pretty much just `-b`, `-O`, and `-OO` - and is is questionable about whether those would apply to code passed through `compile`.

Additionally, passing any options to the compiler through the options box results in `<No output file>`.
The given options are passed after the output file name but before the input file name and interfere with the wrapper script.

Reply:
Setting it to ~~BaseParser~~ makes me able to pass any option to the script. Can we disable that completely (preferably even disable the input field if possible)? AFAIK ~~-O~~ and ~~-OO~~ don't do much at the moment, except of removing doc strings so ~~__doc__~~ for runtime is not set and therefore it doesn't affect our output at all

Reply:
But setting it to `BaseParser` would allow passing every option to the script, correct?. Can we disable that completely (preferably even disable the input field if possible)? AFAIK `-O` and `-OO` don't do much at the moment, except of removing doc strings so `__doc__` for runtime is not set and therefore it doesn't affect our output at all

Reply:
The parser doesn't act as a filter, it just pulls out known arguments and populates them with some help info into the web frontend (click the down arrow to the right of the options box). You can pass whatever arguments you want regardless of the parser used.

Reply:
That's funny, I never noticed the dropdown menu

Reply:
I think this should be removed in favor of using BaseParser for now - maybe indefinitely.

No useful arguments will be parsed out of --help, and if someone does pass them they won't work anyway.

At least with BaseParser they just won't appear in the drop-down.